k8s/client.go

```diff
@@ -1084,6 +1085,23 @@ func (c *Client) GetHelmState(ctx context.Context, namespace string, secretName
 	}, nil
 }
 
+// GetHelmValues is the function for cilium cli sysdump to collect the helm values from the release directly
+func (c *Client) GetHelmValues(_ context.Context, releaseName string) ([]byte, error) {
```

Comment:
follow the conversation https://github.com/liyihuang/cilium-cli/commit/f093eec1c5fee350d8b900d62341b926da4932b8#r123316881

I do see other function dont pass it. I think I can skip it?

https://github.com/liyihuang/cilium-cli/commit/f093eec1c5fee350d8b900d62341b926da4932b8#r123316881

Reply:
Ah, I see. Then this is fine. This appears to be in anticipation of some future method calls which might use the context, which I personally don't agree with including here, but given the style of the file, it's fine to keep like this.